edx_sga/tests/integration_tests.py

```diff
@@ -107,7 +107,7 @@ def make_student(self, block, name, make_state=True, **state):
         item.save()
 
         if answer:
-            student_id = block.student_submission_id(anonymous_id)
+            student_id = block.get_student_item_dict(anonymous_id)
```

Comment:
please add integration test for finalize api to check database behaviour on `finalized` is true or false

Reply:
when i try to run integration tests according to the README (`python manage.py lms --settings=test test edx_sga.tests.integration_tests`), it hangs on this line:

```
nosetests edx_sga.tests.integration_tests --id-file /edx/app/edxapp/edx-platform/.testids/lms/noseids --verbosity=1
Creating test database for alias 'default'...
```

Reply:
i'm going to try to move to the dockerized devstack to get this working. i've been noticing errors that indicate that my VM is out of space, and in general i'm not confident in the machine state. might have better luck with docker. i pushed some changes based on feedback but i'll hold of on the integration test until i can actually get it running